src/components/Tabbar/Tabbar.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { BrowserRouter as Router, Route, Link, Switch } from 'react-router-dom';
+import Profile from '../Profile/Profile';
+import Calculator from '../Calculator/Calculator';
+import Formulasheet from '../Formulasheet/Formulasheet';
+import App from '../../App';
+
+export default function Tabbar() {
+  return (
+    <Router>
+      <div id="site-wrapper">
+        <Switch>
+          <Route exact path="/profile" component={Profile}></Route>
```

Comment:
Can be <Route exact path="/profile" component={Profile} /> instead of open tag, closing tag. (Not important)